src/mypackage/learn.py

```diff
@@ -0,0 +1,5 @@
+#HOW TO DO THAT
+def __init__(self, multiplicand, name='NumpyStyle'):
+    """empty"""
```

Comment:
The docstring of function should describe what the function do. And give users information about the input parameters of the function.